pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminTopicApiTest.java

```diff
@@ -71,6 +75,62 @@ protected void cleanup() throws Exception {
         super.internalCleanup();
     }
 
+    @Test
+    public void testDeleteNonExistTopic() throws Exception {
+        // Case 1: call delete for a partitioned topic.
+        final String topic1 = BrokerTestUtil.newUniqueName("persistent://public/default/tp");
+        admin.topics().createPartitionedTopic(topic1, 2);
+        admin.schemas().createSchemaAsync(topic1, Schema.STRING.getSchemaInfo());
+        Awaitility.await().untilAsserted(() -> {
+            assertEquals(admin.schemas().getAllSchemas(topic1).size(), 1);
+        });
+        try {
+            admin.topics().delete(topic1);
```

Comment:
You can try this way to verify the exception:
```
assertThatThrownBy(()->admin.topics().delete(topic1)).hasMessage("TODO")
```

Reply:
I feel that the current way is clearer and more flexible